test/stores/VDocument.test.ts

```diff
@@ -1088,5 +1088,223 @@ describe('stores', () => {
                 });
             });
         });
+        describe('deleteSelection', () => {
+            it('should delete part of the text within a paragraph', () => {
+                // deleteForward
+                // Forward selection
```

Comment:
Shouldn't these tests be in their respective commands sections ? I mean, `deleteSelection` is not a command per se, it is more like an utils that is used by the other commands. I think the proper deletion of the selection when calling either `deleteForward` or `deleteBackward` is an integral part of the command effect and should be considered as such in the tests.

EDIT: I can argue even more on this: in its current implementation, both `deleteForward` and `deleteBackward` *always* call `deleteSelection`, even in all the previous tests, since all they do is select the right character to delete then call `deleteSelection`, so clearly I see no reason to have these tests separated from the main tests of the command they are using.

Reply:
I initially had them in `deleteBackward` and `deleteForward` but then I found it much more convenient to have them separated because the essence of what they test is so different, and they all share common features. This way it's easier to see if we're missing something.